data/sql/db-world/base/morphsummon.sql

```diff
@@ -33,4 +33,7 @@ INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionTex
 (@MENU_CHOICE, 1, 4, 'Next..', 0, 0, 0, 0, 0, 0, 0, '', 0, 0),
 (@MENU_CHOICE, 2, 4, 'Previous..', 0, 0, 0, 0, 0, 0, 0, '', 0, 0);
 
+DELETE FROM `acore_world`.`creature_template_model` where `CreatureID` = @ENTRY;
+INSERT INTO `acore_world`.`creature_template_model` (`CreatureID`, `Idx`, `CreatureDisplayID`, `DisplayScale`, `Probability`, `VerifiedBuild`) VALUES (@ENTRY, 0, @MODELID, 1, 1, 12340);
```

Comment:
`acore_world` is not necessary
```suggestion
DELETE FROM `creature_template_model` where `CreatureID` = @ENTRY;
INSERT INTO `creature_template_model` (`CreatureID`, `Idx`, `CreatureDisplayID`, `DisplayScale`, `Probability`, `VerifiedBuild`) VALUES (@ENTRY, 0, @MODELID, 1, 1, 12340);
```